README.md

```diff
@@ -34,38 +40,35 @@ The MSAL preview library for iOS and macOS gives your app the ability to begin u
-
-
-## Installation
+## Optional: Register your App  
 
-We use [Carthage](https://github.com/Carthage/Carthage) for package management during the preview period of MSAL. This package manager integrates very nicely with XCode while maintaining our ability to make changes to the library. The sample is set up to use Carthage.
+The app comes pre-configured for testing.  If you would like to register your own app, please follow 
+the steps below. 
 
-##### If you're building for iOS, tvOS, or watchOS
+You will need to have a native client application registered with Microsoft using the 
+[App Registration Portal](https://apps.dev.microsoft.com/portal/register-app?appType=mobileAndDesktopApp&appTech=android). 
 
-1. Install Carthage on your Mac using a download from their website or if using Homebrew `brew install carthage`.
-1. We have already created a `Cartfile` that lists the MSAL library for this project on Github. We use the `/dev` branch.
-1. Run `carthage update`. This will fetch dependencies into a `Carthage/Checkouts` folder, then build the MSAL library.
-1. On your application targets’ “General” settings tab, in the “Linked Frameworks and Libraries” section, drag and drop the `MSAL.framework` from the `Carthage/Build` folder on disk.
-1. On your application targets’ “Build Phases” settings tab, click the “+” icon and choose “New Run Script Phase”. Create a Run Script in which you specify your shell (ex: `/bin/sh`), add the following contents to the script area below the shell:
+To create an app,  
+1. Click the `Add an app` button inside the *Converged Apps* section.
 
-  ```sh
-  /usr/local/bin/carthage copy-frameworks
-  ```
+2. Name your app and select `Create`. 
+    - After the app is created, you'll land on your app management page. 
 
-  and add the paths to the frameworks you want to use under “Input Files”, e.g.:
+3. Click `Add Platform`, then select `Native Application`. 
+    - The Redirect URI produced is needed when making Auth requests. If you're using MSAL, it will
+    be automatically constructed by the library. 
 
-  ```
-  $(SRCROOT)/Carthage/Build/iOS/MSAL.framework
-  ```
-  This script works around an [App Store submission bug](http://www.openradar.me/radar?id=6409498411401216) triggered by universal binaries and ensures that necessary bitcode-related files and dSYMs are copied when archiving.
+4. Hit the `Save` button. 
 
-With the debug information copied into the built products directory, Xcode will be able to symbolicate the stack trace whenever you stop at a breakpoint. This will also enable you to step through third-party code in the debugger.
+## Installation
 
-When archiving your application for submission to the App Store or TestFlight, Xcode will also copy these files into the dSYMs subdirectory of your application’s `.xcarchive` bundle.
+Load the podfile using cocoapods. This will create a new XCode Workspace you will load.
 
+```
+$ pod install
+...
+$ open QuickStart.xcworkspace
```

Comment:
I have not found QuickStart.cxworkspace. Would it be **project.xcworkspace** ?

Reply:
It is `MSALiOS.xcworkspace` which I updated. Another cut/paste error.